modules/localgov_campaigns_paragraphs/js/localgov-tabs.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * @file
+ * Tabs behaviour.
+ */
+
+(function () {
+  Drupal.behaviors.tabs = {
+    /**
+     * Attach Tabs behaviour.
+     *
+     * @param {object} context
+     *   DOM object.
+     */
+    attach: function attach(context) {
+      var tabs = context.querySelectorAll('[data-tabs]');
+
+      for (var i = 0; i < tabs.length; i++) {
+        this.init(tabs[i], i);
+      }
+    },
+
+    /**
+     * Initialise tabs.
+     *
+     * @param {HTMLElement} tabs
+     *   Tabs element.
+     * @param {integer} index
+     *   Current index.
+     */
+    init: function init(tabs, index) {
+      var _this = this;
+      var tabPanels = tabs.querySelectorAll('.tab-panel');
+      var tabsInitialisedClass = 'tabs--initialised';
+      var breakpoint = tabs.dataset.accordionTabsSwitch || null;
+      var tabId = index;
+
+      var create = function create() {
+        if (!tabs.classList.contains(tabsInitialisedClass)) {
+          var _ret = function () {
+            var tabPanelsNumber = tabPanels.length;
+
+            // Only initialise tabs if there are at least 2 panels.
+            if (tabPanelsNumber < 2) {
+              return {
+                v: void 0
```

Comment:
MDN has something interesting to [say about the use of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void#Description) ```void 0```.

Reply:
This is actually how ES6 code is compiled. 
So this:
```
 if (tabPanelsNumber < 2) {
   return;
 }
```
becomes this:
```
if (tabPanelsNumber < 2) {
  return {
    v: void 0
  };
}
```

Reply:
Thanks for the clarification Gosia :)